crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -233,15 +251,26 @@ pub fn extract_meshes(
             &Handle<Mesh>,
             Option<With<NotShadowReceiver>>,
             Option<With<NotShadowCaster>>,
+            Has<NoAutomaticBatching>,
```

Comment:
Though `NoAutomaticBatching` isn't automatically added to meshes with skin or morphs

Reply:
I solved it in a way that I don't really like where the automatic_batching field in RenderMeshInstance is overridden in skin and morph extraction systems. The reason I don't really like it is that it serialises their execution. But probably skinned meshes aren't going to be in massive numbers so it's perhaps not too a big a deal. Otherwise, yes, we could identify skinned meshes in `PostUpdate` or something, and make sure they have `NoAutomaticBatching` on them.